cypress/utils/cypress_react_test.js

```diff
@@ -0,0 +1,81 @@
+// This file is taken from https://github.com/bahmutov/cypress-react-unit-test
+// The only difference is it references React 15 instead of React 16
```

Comment:
Like the comment says, I copied the [cypress-react-unit-test](https://github.com/bahmutov/cypress-react-unit-test) library here to get it to work...forking the repo might be smarter, but it's pretty small.